test/ng/compileSpec.js

```diff
@@ -3137,6 +3137,54 @@ describe('$compile', function() {
     });
 
 
+    it('should support custom start interpolation symbol, even when `endSymbol` doesn\'t change',
```

Comment:
What @gkalpak means is that these test that the functionality works even if you change **only one** of `startSymbol` and `endSymbol`